packages/unity-interface/dcl.ts

```diff
@@ -90,9 +89,10 @@ async function startGlobalScene(cid: string, title: string, fileContentUrl: stri
     mappings: []
   })
 
-  const worker = loadParcelScene(scene, undefined, true)
+  loadParcelScene(scene, undefined, true)
 
-  await ensureUiApis(worker)
+  // TODO
+  // await ensureUiApis(worker)
```

Comment:
`ensureUiApis ` check if `SocialController` host API exists. And looks like you've already created. So maybe this is old?